Sources/Markdown/Walker/Walkers/HtmlFormatter.swift

```diff
@@ -0,0 +1,319 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+/// Options given to the ``HtmlFormatter``.
+public struct HtmlFormatterOptions: OptionSet {
```

Comment:
nit: can we uppercase HTML here and in the other type names? Apple's SDK symbols uses the same text casing for all letters in acronyms and initialisms. For example[`XMLParser`][1], [`HTTPURLResponse`][2], and [`loadHTMLString(_:baseURL:)`][3].

[1]: https://developer.apple.com/documentation/foundation/xmlparser
[2]: https://developer.apple.com/documentation/foundation/httpurlresponse
[3]: https://developer.apple.com/documentation/webkit/wkwebview/1415004-loadhtmlstring

Reply:
Done! I wasn't sure about it when i initially wrote the code, but i appreciate having the examples linked.